demisto_sdk/commands/validate/validators/PB_validators/PB108_is_valid_task_id.py

```diff
@@ -0,0 +1,65 @@
+from __future__ import annotations
+
+from typing import Iterable, List
+
+from demisto_sdk.commands.common.tools import is_string_uuid
+from demisto_sdk.commands.content_graph.objects.base_playbook import TaskConfig
+from demisto_sdk.commands.content_graph.objects.playbook import Playbook
+from demisto_sdk.commands.validate.validators.base_validator import (
+    BaseValidator,
+    ValidationResult,
+)
+
+ContentTypes = Playbook
+
+
+class IsValidTaskIdValidator(BaseValidator[ContentTypes]):
+    error_code = "PB108"
+    description = "Validate that the task ID and the 'id' under the 'task' field are from UUID format"
+    rationale = ""
+    error_message = (
+        "On task: {0},  the field 'taskid': {1} and the 'id' under the 'task' field: {2}, must be from "
+        "uuid format."
+    )
+    related_field = "taskid"
+    is_auto_fixable = False
+    # expected_git_statuses = [GitStatuses.ADDED, GitStatuses.MODIFIED]
+
+    def is_valid(self, content_items: Iterable[ContentTypes]) -> List[ValidationResult]:
+        """Check whether all playbook tasks has valid taskid and the 'id' under the 'task' field is valid as well.
+        Args:
+            - content_items (Iterable[ContentTypes]): The content items to check.
+        Return:
+            - List[ValidationResult]. List of ValidationResults objects.
+        """
+        results: List[ValidationResult] = []
+        for content_item in content_items:
+            invalid_tasks = self.invalid_tasks(content_item.tasks)
+            for task_id in invalid_tasks:
+                taskid = invalid_tasks[task_id].taskid
+                inner_id = invalid_tasks[task_id].task.id
+                results.append(
+                    ValidationResult(
+                        validator=self,
+                        message=self.error_message.format(task_id, taskid, inner_id),
+                        content_object=content_item,
+                    )
+                )
+        return results
```

Comment:
Let's make sure to return only one ValidationResult per contentitem.
Also, let's only mention the relevant fields that are not in the right format.
Consider alter **invalid_tasks** to return a dict of {"id": [malformed fields]}.